README.md

```diff
@@ -23,13 +37,29 @@ options:
   -t, --border_type     Border Type: p for polaroid, s for small, m for medium, l for large, i for instagram (default: s)
   --include             File patterns to include (default: *.jpg *.jpeg *.png, *.JPG, *.JPEG, *.PNG)
   --exclude             File patterns to exclude (default: *_border*)
+
 Made for fun and to solve a little problem.
 ```
 
 ---
 
 > Note: This is a hacked together little script. Use at your own peril...
 
+## osx_services
+
+Adds quick actions to you OSX menu for quick deployment of tool.
+
+## Fonts
+
+The repo comes with [Roboto](https://fonts.google.com/specimen/Roboto) (Regular, Medium & Bold).
+
+```photoborder/fonts```
+
+Should you wish to use another font you will need to manually update ```text.py```
```

Comment:
Alternatively, now that I think of it, it could be moved to a parameter with appropriate bounds checking to reduce complexity:

parser.add_argument('-f', '--font', default='Roboto-Regular.ttf', help='Font file in fonts directory')
parser.add_argument('-fb', '--fontbold', default='Roboto-Medium.ttf', help='Bold font file in fonts directory')

Reply:
Let me take a look. I did think of an environment file but with time and the need to research how to do it (my coding experience is limited, but I enjoy learning) I was going to resolve at a later date. Although, I'd be happy to pull that timeline forward and hold this PR open for a bit.

I'll have a go and see how it goes based on the guidance above. 

👍

Reply:
I almost think parameters might be a better option? It's less initial configuration for the user, and easy to pass in new options and have sane defaults. 

What do you think?

Reply:
So I've created it as you suggested (feels the best method) and added a validation font function to gracefully exit if the font is not available in the directory. 

I did utilise **CoPilot** for support as I got stuck on passing arguments to the main function and had it incorrectly defined. I had global definitions for font/bold font in `text.py` and that clearly wasn't working. Shifting to `main.py` under the main function fixed that with the parameters modified in `border.py`,` text.py `& `main.py`

```
font_path = os.path.join(FONTDIR, args.font)
bold_font_path = os.path.join(FONTDIR, args.fontbold)
```
Figured it out with help. The `README.md` has been updated and a small tweak to `.gitignore` too.

I won't be offended if it needs to be tidied. However, it does work as intended.

👍

Reply:
Good stuff, thanks for this. I just streamlined the error reporting a little bit as there were some duplicates, and removed some redundant code. I also added python logging so we can use that instead of print statements from now on. 

I'll wait for you to review before I merge into main.